tests/tizen_capi/unittest_tizen_sensor.cc

```diff
@@ -18,6 +18,20 @@
 #include <gst/gst.h>
 #include <nnstreamer.h>
 #include <nnstreamer-capi-private.h>
+#include <unittest_util.h>
+
+static const uint TEST_TIME_OUT_TIZEN_SENSOR = 10000; /* timeout occur after 10 seconds */
+
+#define wait_for_start(handle, state, status) \
+do {\
+  int counter = 0;\
+  while (state != ML_PIPELINE_STATE_PLAYING && counter < 1000) {\
+    g_usleep (10000);\
+    counter ++;\
+    status = ml_pipeline_get_state (handle, &state);\
+    EXPECT_EQ (status, ML_ERROR_NONE);\
+  }\
```

Comment:
What if it becomes PAUSED->PLAYING-(completed)->EOS during g_usleep?

Reply:
As I know pipeline maintains playing state even if EOS signal is emitted.

Reply:
Nah.. never mind this. With large enough src (num-buffers 30 @ 60 FPS), you don't need to worry about this.